cloudgenix_config/pull.py

```diff
@@ -1619,7 +1667,7 @@ def pull_config_sites(sites, output_filename, output_multi=None, passed_sdk=None
                 config_yml.write("# Created at {0}\n".format(datetime.datetime.utcnow().isoformat()+"Z"))
                 if sdk.email:
                     config_yml.write("# by {0}\n".format(sdk.email))
-
+                config_yml.write("# MANDATORY: Add 'parent_type: bypasspair field for 'ion 9000' if parent/source_interface etc. is bypasspair in range 12-16\nElse we may configure wrong interface\n")
```

Comment:
We should print this message when running the pull_site too. 

Update the message to:
# Note: For ION 9Ks interface configuration, if the source_interface or parent_interface is a bypasspair port, add the attribute "parent_type": bypasspair.  \nIf this field is not specified, the cloudgenix_config utility will assume the parent interface is of type "port".

Reply:
Updated in the latest push